### DISCUSSION TITLE
[QUESTION] Is it possible to enable ACL for NFS in longhorn?

### DISCUSSION THREAD

**Discussion by MohammedNoureldin** [longhorn/longhorn#6823] [Ideas]
Is it possible to enable ACL when using longhorn RWX?

My exact use-case is in sharing some info between Samba domain controllers. When I try to provision a new domain controller, it requires for the SysVol to be on a volume that supports ACL, which seems to not be the case with RWX.

> ERROR: Provision failed - ProvisioningError: Your filesystem or build does not support posix ACLs, which s3fs requires.  Try the mounting the filesystem with the 'acl' option.

**Comment by c3y1huang** [comment]
As far as I know, Longhorn RWX volumes do not currently support ACL.

**Reply by MohammedNoureldin** [reply]
Thank you for your reply. Can't this be considered as a feature request?

**Reply by c3y1huang** [reply]
Sure. https://github.com/longhorn/longhorn/issues/6825

**Comment by derekbit** [comment]
@MohammedNoureldin 
Can you provide a manifest for the reproducing the issue?

**Reply by MohammedNoureldin** [reply]
@derekbit the current manifest is too complex for this. I will try to make it as simple as possible and then share it.
